modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -509,6 +510,9 @@ public void updateCodegenPropertyEnum(CodegenProperty var) {
 
     @Override
     public CodegenParameter fromRequestBody(RequestBody body, Set<String> imports, String bodyParameterName) {
+        if (StringUtils.isEmpty(bodyParameterName)) {
```

Comment:
@spacether can you please provide a spec that results in an empty body parameter name?

Reply:
This happens when the body parameter is $refed and doesn't use x-codegen-request-body-name.
Our one higher invocation for that use case is [here](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L3691)
This code path is already tested with our v3 sample spec with these examples:
- [PUT /pet](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/python-experimental/petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml#L71)
- POST /pet
- POST /store/order
- POST /user
- POST /user/createWithArray
- and others
Because we already have a lot of coverage here in my opinion adding another endpoint here doesn't help us. WHat do you think?

Reply:
The name change code has been reverted s it's not the main focus of this PR